website/docs/modules/ROOT/partials/Publishing_Footer.adoc

```diff
@@ -42,6 +95,84 @@ export MILL_PGP_SECRET_BASE64=...
 export MILL_PGP_PASSPHRASE=...
 ----
 
+=== Upload Public Key for Artifacts Verification
```

Comment:
This should be at the start of the section since it is what people will need to do first